packages/compass-aggregations/package.json

```diff
@@ -96,7 +96,8 @@
     "react-redux": "^8.1.3",
     "redux": "^4.2.1",
     "redux-thunk": "^2.4.2",
-    "semver": "^5.7.2"
+    "semver": "^5.7.2",
+    "uuid": "^8.3.2"
```

Comment:
Would we potentially want to move away from `uuid` as a separate dependency and use `new UUID().toString()` from `bson` instead? We currently have a couple of uses of the latter, and one usage of `uuid` in compass-connections

Reply:
Good thinking! I'll do a follow up pr and remove it fully from the code base.